src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift

```diff
@@ -384,6 +386,14 @@ class HomeTableViewController: UITableViewController, NavigationBarOpacityDelega
 			UINib(nibName: String(describing: HomeRiskTableViewCell.self), bundle: nil),
 			forCellReuseIdentifier: String(describing: HomeRiskTableViewCell.self)
 		)
+		tableView.register(
+			UINib(nibName: String(describing: HomeVaccinationTableViewCell.self), bundle: nil),
+			forCellReuseIdentifier: String(describing: HomeVaccinationTableViewCell.self)
```

Comment:
we have a protocol for the reuseIdentifier ˚ReuseIdentifierProviding˚, if the cell fulfills it you can use .reuseIdentifier

Reply:
Is this new?